hadoop-ozone/dist/src/main/compose/ozone-ha/docker-config

```diff
@@ -23,6 +23,9 @@ OZONE-SITE.XML_ozone.om.address.omservice.om2=om2
 OZONE-SITE.XML_ozone.om.address.omservice.om3=om3
 OZONE-SITE.XML_ozone.om.ratis.enable=true
 
+OZONE-SITE.XML_ozone.om.http-address=0.0.0.0:9874
+OZONE-SITE.XML_ozone.scm.http-address=0.0.0.0:9876
+
```

Comment:
These two definitions are unnecessary.  With ports 9874 and 9876 exposed in docker containers, by default the om and scm webservers serve the ports.  Works without.

Reply:
You are right, I'll remove them.